dist/index.html

```diff
@@ -1,34 +1,40 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Art Gallery</title>
 <script defer src="main.js"></script></head>
-    <body>
-        <!-- ***********************  PAGE OVERALL CONTENT WRAPPER *********************** -->
-        <div class="container">
-      
-          <!-- ***********************  PAGE HEADER *********************** -->
-          <header class="header"></header>
-          <!-- ***********************  END OF PAGE HEADER *********************** -->
-      
-          <!-- ***********************  PAGE MAIN *********************** -->
-          <main class="main">
-      
-          </main>
-          <!-- ***********************  END OF PAGE MAIN *********************** -->
-      
-          <!-- ***********************  PAGE FOOTER *********************** -->
-          <footer class="footer">
-            <div class="footer__wrapper">
-             
-            </div>
-          </footer>
-          <!-- ***********************  END OF PAGE FOOTER *********************** -->
-      
+<body>
+  <div class="containerCommentsPopUp">
+  </div>
+  <div class="reservationPopup">
+  </div>
+    <header>
+      <div>
+        <img
+          src="https://raw.githubusercontent.com/fmanimashaun/artwork-listing-app/combined_soln/src/img/logo.png"
+          class="home-logo"
+          alt=""
+          sizes=""
+          srcset=""
+        />
+        <span class="fd-number">Gallery (<span class="fd-count">0</span>)</span>
+        <span class="fd-categories">Painting</span>
+        <span class="fd-categories">Shop</span>
+        <div class="artworks">
+          <h1>Artworks</h1>
         </div>
-        <!-- ***********************  END OF PAGE OVERALL CONTENT WRAPPER *********************** -->
-      </body>
-</html>
\ No newline at end of file
+      </div>
+    </header>
+    <main>
+      <section class="gallery"><!--gallery div --></section>
+      <div class="grid-container"></div>
+    </main>
+    <footer>
+      <p>Created by Microverse under CC license</p>
+    </footer>
+  </body>
```

Comment:
- [ ] Good job making the footer with the right content. 👏 . However, you'll notice it's sectionalized with a line that I believe you missed. Kindly include it to give your web app a more appealing and clear-cut view from a user perspective 👍 
- [ ] Also, please adjust the font to  make it more visible 
# Your view:
![image](https://user-images.githubusercontent.com/50990925/218246429-dbb40f69-f482-4d1b-a8e4-1e0474e2a4fc.png)

# Wireframe:
![image](https://user-images.githubusercontent.com/50990925/218246444-7810aad6-0b99-4d00-bb6a-90bf7fefa4cb.png)

Reply:
Well resolved ✔️